### PR TITLE
Release v0.19.3.1

### PR DESCRIPTION
**Pull request by vishh**
Cherrypick of #960

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
 # Changelog
 
+### 0.19.3.1 (2015-12-03)
+- Add a flag `--no-systemd` to avoid using systemd with docker.
```
Comment (dchen1107) [inline CHANGELOG.md]: Can you also have the issue # here? The release note you put here is pretty confusion.

**Comment by dchen1107** [comment]
small nits.

LGTM

File: CHANGELOG.md (hunk 1 above)
Comment (vishh) [inline CHANGELOG.md]: Mentioned the original issue number. PTAL

**Comment by vishh** [comment]
Jenkins failed with a gcloud error. Tests passed. Self merging this PR for urgency.

**Pull request merged**